src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-23931"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {}
```

Comment:
> the netcoreapp* TFM is only be paired with Microsoft.NETCore.App meta-package.

Who makes/enforces that rule?  Or is it not an official rule?  @anurse told me you can use netcoreapp TFM, but reference NETStandard.Library.